src/XrdCrypto/XrdCryptosslCipher.hh

```diff
@@ -39,8 +39,6 @@
 #include <openssl/evp.h>
 #include <openssl/dh.h>
 
-#define kDHMINBITS 512
```

Comment:
This was needed by the commented out code, so maybe move it there. Otherwise, if/when we uncomment the code, it will fail to compile.

Reply:
Good catch.  I decided to keep it where it was but instead provide context/explanation.